Command/IndexImportCommand.php

```diff
@@ -43,6 +43,12 @@ protected function configure()
                 InputOption::VALUE_REQUIRED,
                 'Set bulk size for import',
                 1000
+            )
+            ->addOption(
+                'gzip',
+                null,
```

Comment:
You can use `z`shortcut, it should be available. When we were discussing about shortcuts with multiple letters, the problem was that one of the letter was reserved, that's why there was a mistake with multiple. Since `z` is free it could be added. btw `z` in most of the unix commands means compressing.